assemble/pom.xml

```diff
@@ -323,5 +323,67 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>hadoop2</id>
+      <activation>
+        <property>
+          <!-- Hack around activeByDefault=true not working as we'd want -->
+          <name>!hadoop.profile</name>
+        </property>
+      </activation>
+      <properties>
+        <artifactList>commons-math3,commons-vfs2,gson,guava,htrace-core,javax.servlet-api,jcommander,jetty-http,jetty-io,jetty-security,jetty-server,jetty-servlet,jetty-util,jline,libthrift,protobuf-java,slf4j-api,slf4j-log4j12</artifactList>
+        <assemblyDescriptor>src/main/assemblies/binary-release.xml</assemblyDescriptor>
+      </properties>
+    </profile>
+    <profile>
+      <id>hadoop3</id>
+      <activation>
+        <property>
+          <name>hadoop.profile</name>
+          <value>3</value>
+        </property>
+      </activation>
+      <properties>
+        <artifactList>commons-cli,commons-codec,commons-collections,commons-configuration,commons-io,commons-lang,commons-logging,commons-math3,commons-vfs2,gson,guava,htrace-core,htrace-core4,javax.servlet-api,jcommander,jetty-http,jetty-io,jetty-security,jetty-server,jetty-servlet,jetty-util,jline,libthrift,protobuf-java,slf4j-api,slf4j-log4j12</artifactList>
+        <assemblyDescriptor>src/main/assemblies/binary-release-hadoop3.xml</assemblyDescriptor>
+      </properties>
+      <!-- When using Hadoop 3 with the shaded jars, we need to provide
+ 					 the jars we need instead of relying on them externally. -->
```

Comment:
Do these additional bundled jars affect the LICENSE/NOTICE files of our binary tarball in any way?

Reply:
Sadly, probably.

Reply:
Turns out we don't need any additions. The only artifact with an entry to NOTICE is about a file that's only in the test jar (and thus we don't need to include).